plugins/dbviewer/api/api.js

```diff
@@ -257,6 +257,25 @@ var common = require('../../../api/utils/common.js'),
                 return callback(false);
             }
         }
+        /**
+        * Get aggregated result by the parameter on the url
+        * @param {string} collection - collection will be applied related query
+        * @param {object} match - match object for aggregation
+        * @param {object} group - group object for aggregation
+        * */
+        function aggregate(collection, match, group) {
+            dbs[dbNameOnParam].collection(collection).aggregate([
+                { $match: match },
+                { $group: group }
```

Comment:
This is very limited aggregation
It would be better to accept single parameter `pipeline` or the same `aggregation` param you have, which will array of commands and pass them to aggregate method directly